spec/fixtures/example_markdown.md

```diff
@@ -0,0 +1,22 @@
+## Test Valid External Links
+
+[valid external link](http://example.org)
+[valid external link with title](http://example.org "example title")
+[valid external link with alt|example alt](http://example.org)
+[valid external link with alt and title|example alt](http://example.org "example title")
+
+## Test Valid Mailto
+
+[valid mailto](mailto:admin@example.org)
+
+## Test Valid Image URLs
+
+![valid image](https://en.wikipedia.org/wiki/File:Example.jpg)
+![valid image with title](https://en.wikipedia.org/wiki/File:Example.jpg "example image title")
+![valid image with alt|example image alt](https://en.wikipedia.org/wiki/File:Example.jpg)
+![valid image with alt and title|example image alt](https://en.wikipedia.org/wiki/File:Example.jpg "example image title")
```

Comment:
I feel a *little* weird depending on something outside our control... but I suppose it is more stable than any images on our own sites.